webcompat/static/js/lib/bugform.js

```diff
@@ -573,6 +605,136 @@ function BugForm() {
     });
   };
 
+  // functions for Image Editor
+  this.openImageEditor = function() {
+    //var doc = global.document;
+    var canvas = document.createElement('canvas');
+    var ctx = canvas.getContext('2d');
+    var img = new Image();
+    canvas.id = "mCanvas";
+    canvas.width = 500;
+    canvas.height = 500;
+    img.src = this.dataURI;
+
+    var modalContent = document.querySelector('.modal-content');
+
+    this.editImageModal.removeClass("is-hidden");
+    modalContent.appendChild(canvas);
+
+    // TODO ask Mike if there is a better way to share this globally, like this.ctx
+    globalCtx = ctx;
+    globalCtx.drawImage(img, 0, 0);
+
+    // TODO ask Mike if there is a better way
+    // having to define the canvas event listeners here because at time
+    // of init definition, the canvas does not exist
```

Comment:
It's fine to add handlers here, it doesn't have to be on init. Just before we expect to do anything useful with them.